### PR TITLE
Graph fix

### PR DESCRIPTION
**Pull request by devsdevsdevs**
**Summary**
The graph components had a few issues that needed to resolved. Within this repository, the `quarter_day_ranges.json` file needed to be updated, for the data we currently have in the database. There were also small bugs to be fixed affecting crashing/loading. Outside of the repository, I discovered [issues](https://github.com/icssc-projects/SOCSpider/commit/9d9716a28266e35739c38d9f5361b25fc1e6ca8c) with SOCSpider and fixed/deployed them. I also [updated ](https://github.com/icssc-projects/antalmanac-backend/commit/f3a86836850f10b107c0bd3411d72abe333055e9) the backend to correct a schema issue that led to graphs not loading on the frontend.

cap1
cap2

**Test Plan**
- If course was not offered in a quarter, show "This course wasn't offered in ..." text
- Shows rows in modal only for courses with > 0 units
- Shows proper date range for the quarter in the graph
- Hovering over the graph shows popup with data for a specific date
- Course data in row is accurate, loads for quarter selected

**Issues**
Addresses #44

### CODE REVIEW

File: client/src/components/EnrollmentGraph/GraphModalContent.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ class GraphModalContent extends PureComponent {
                 <FormControl fullWidth>
                     <InputLabel>Term</InputLabel>
                     <Select value={this.state.pastTerm} onChange={this.handleChange}>
-                        <MenuItem value={'2020 Spring'}>2020 Spring Quarter</MenuItem>
+                        <MenuItem value={'2021 Winter'}>2021 Winter Quarter</MenuItem>
```
Comment (ChaseC99) [inline client/src/components/EnrollmentGraph/GraphModalContent.js:100]: Why do we only have data for Winter 2021?

**Review by ChaseC99 — APPROVED** [review]
Graphs are looking good! Solid work 👍

**Comment by braedons** [comment]
Sorry for the incredible delay, but I tried for CS 121 and DRAMA 20C, and I get an empty graph. 
image

After about a minute, I get this error:
image

File: client/src/components/EnrollmentGraph/GraphModalContent.js (hunk 1 above)
Comment (devsdevsdevs) [reply client/src/components/EnrollmentGraph/GraphModalContent.js:100]: I haven't uploaded the data for Fall 2020. The data was collected locally and needs to be moved.

**Comment by devsdevsdevs** [comment]
@braedons I think you need to update the backend submodule since I also made updates to it

**Pull request merged**